corehq/apps/locations/util.py

```diff
@@ -429,3 +430,10 @@ def get_location_type(domain, location, parent, loc_type_string, exception, is_n
     return loc_type_obj
 
 # ---
+
+
+# Checks if a location type has any users assigned to it's related locations
+def does_location_type_have_users(loc_type):
+    location_ids = list(SQLLocation.objects.filter(location_type=loc_type).values_list('location_id', flat=True))
+    users = user_ids_at_locations(location_ids)
+    return bool(users)
```

Comment:
This will fetch all IDs of matching docs, which is unnecessary - this should be a smidge more efficient:
```suggestion
    return bool(UserES().location(location_ids).count())
```
Looks like we could do [even better](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-your-data.html#quickly-check-for-matching-docs), but I don't think we have the wrappers for that yet.